lib/propolis/src/hw/virtio/p9fs.rs

```diff
@@ -456,15 +467,21 @@ impl HostFSHandler {
         // mode: u32,
         let mode = metadata.mode();
         // uid: u32,
-        //let uid = metadata.uid();
         let uid = 0; //squash for now
                      // gid: u32,
                      //let gid = metadata.gid();
         let gid = 0; //squash for now
                      // nlink: u64,
         let nlink = metadata.nlink();
         // rdev: u64,
-        let rdev = metadata.rdev();
+        let rdev =
+            if metadata.is_file() || metadata.is_dir() || metadata.is_symlink()
```

Comment:
you'd mentioned `p9fs parlance` above, is there a spec or do you mean parlance as in the implementation of p9fs somewhere else? I found https://ericvh.github.io/9p-rfc/rfc9p2000.html#anchor18 but it's not so helpful here :)

Reply:
@dancrossnyc mentioned on the side that what I'm looking for here is 9p2000.L, or https://github.com/chaos/diod/blob/master/protocol.md